pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -192,6 +196,8 @@
 
 public class PersistentTopic extends AbstractTopic implements Topic, AddEntryCallback {
 
+    public static final long NOT_AVAILABLE_YET = -1;
```

Comment:
nit:
Maybe we can return `-1` in the method directly.
It's a little confused the NonPersistentTopic will also use this static field

Reply:
You are correct. I inlined it.